packages/matter.js/src/codec/BtpCodec.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ByteArray, Endian } from "../util/ByteArray.js";
+import { DataReader } from "../util/DataReader.js";
+import { DataWriter } from "../util/DataWriter.js";
+
+export interface BtpHandshakeRequest {
+    version: ByteArray,
+    attMtu: number,
+    clientWindowSize: number,
+}
+
+export interface Header {
```

Comment:
better als call BtpHeader to be naming-wise consistent?

Reply:
Updated